src/backend/wsl.ts

```diff
@@ -288,6 +285,8 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
     if (!await this.isDistroRegistered()) {
       throw new Error(`Error registering WSL2 distribution`);
     }
+
+    await this.initDataDistribution();
```

Comment:
This needs to always be called (we can have the main distro registered without the data distro existing).